src/speed_lookup_bindings.hpp

```diff
@@ -3,6 +3,7 @@
 #include <nan.h>
 
 #include "segment_speed_map.hpp"
+#include "types.hpp"
```

Comment:
Ah, ok, then the include should be in `speed_lookup_bindings.cpp`, not the `.hpp` file - the general rule is to only include headers where they're needed.  `speed_lookup_bindings.hpp` doesn't use `INVALID_SPEED`, so it doesn't need the header.